src/protobuf/sabre_pb2/sabre_payload.pb.go

```diff
@@ -0,0 +1,1065 @@
+// Code generated by protoc-gen-go. DO NOT EDIT.
```

Comment:
For any auto-generated files, it would be helpful to include documentation about how the files are generated. In non-Go repos we don't allow auto-generated files, but Go requires it. Still, we need instructions to be able to repeat the process. This could go in a BUILD.md file, or the README.md directly.

Long-term, we will have to determine best practice for auto-generated files with respect to commits; I suspect best practice is to separate them out to separate commits so we get less conflicts cherry-picking commits around branches.